package.json

```diff
@@ -30,7 +30,9 @@
     "moduleNameMapper": {
       "^js(.*)$": "<rootDir>/src/js$1",
       "^dojo(.*)$": "<rootDir>/__mocks__/arcgis.js",
-      "^esri(.*)$": "<rootDir>/__mocks__/arcgis.js",
+      "esri/request": "<rootDir>/__mocks__/MapView.js",
+      "esri/views/MapView": "<rootDir>/__mocks__/MapView.js",
+      "esri/Map": "<rootDir>/__mocks__/Map.js",
       "^dijit(.*)$": "<rootDir>/__mocks__/arcgis.js",
```

Comment:
I wonder if we can still leave the esri mapper in place but underneath those. Something like this:

```
"esri/request": "<rootDir>/__mocks__/MapView.js",
"esri/views/MapView": "<rootDir>/__mocks__/MapView.js",
"esri/Map": "<rootDir>/__mocks__/Map.js",
"^esri(.*)$": "<rootDir>/__mocks__/arcgis.js"
```

I can test this when I get home, but what I am thinking is this way if a developer wants to test MapView and needs a custom mock for that, They can provide a custom Mock for that without having to specify Mocks for all other esri files used in the app. That way, `"^esri(.*)$": "<rootDir>/__mocks__/arcgis.js"`, could catch all esri modules except one's provided above, kind of like a catch everything not remapped above clause.

I don't know if Jest works like this but I can test it out later and see.